python/chronos/src/bigdl/chronos/forecaster/__init__.py

```diff
@@ -35,8 +35,10 @@
 except:
     warnings.warn("Please install `tensorflow>2.0.0` to use MTNetForecaster.")
 try:
-    import prophet
-    prophet_available = True
+    dependency_list = subprocess.getoutput(["pip freeze | grep prophet"])
```

Comment:
`pip` could be not available in some users' environment (e.g. they use `pip3` or something like it).